rts/lra/lra-coordinator/pom.xml

```diff
@@ -61,6 +61,10 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-jwt</artifactId>
```

Comment:
I assume the scope should be either `provided` or `test`

Reply:
I will update it as `test`